### ISSUE TITLE
Missing dependencies?

### ISSUE THREAD

**Issue by jhfrontz** [niw/HapticKey#37]
I'm trying to build after cloning the repo -- I've got Xcode 11.3.1.  However, when I try to `make`, I get:

```
$ make
scripts/pod install
scripts/pod: line 5: /Users/jhf/stuff/HapticKey/scripts
/Users/jhf/stuff/HapticKey/scripts/exec_with_gems: No such file or directory
scripts/pod: line 5: exec: /Users/jhf/stuff/HapticKey/scripts
/Users/jhf/stuff/HapticKey/scripts/exec_with_gems: cannot execute: No such file or directory
make: *** [Pods/Manifest.lock] Error 126
```

Is there some other dependency that I'm missing?

**Comment by niw** [comment]
@jhfrontz `scripts/exec_with_gems` should exist (See https://github.com/niw/HapticKey/blob/master/scripts/exec_with_gems,) probably your working directory is not clean.

**Comment by jhfrontz** [comment]
@niw that's the weird thing-- it does exist:

```
$ find scripts -type f
scripts/pod
scripts/sparkle_appcast
scripts/exec_with_gems
```

If I run the `pods` script directly:
```
$ bash -x  scripts/pod install 
+++ dirname scripts/pod
++ cd -P scripts
++ pwd
+ readonly 'current_dir=./scripts
/Users/jhf/stuff/HapticKey/scripts'
+ current_dir='./scripts
/Users/jhf/stuff/HapticKey/scripts'
+ exec './scripts
/Users/jhf/stuff/HapticKey/scripts/exec_with_gems' pod install
scripts/pod: line 5: /Users/jhf/stuff/HapticKey/scripts
/Users/jhf/stuff/HapticKey/scripts/exec_with_gems: No such file or directory
scripts/pod: line 5: exec: /Users/jhf/stuff/HapticKey/scripts
/Users/jhf/stuff/HapticKey/scripts/exec_with_gems: cannot execute: No such file or directory
```

Aha!  I think it is this:  https://github.com/devrandom/gitian-builder/pull/202/files#r249558443
(https://unix.stackexchange.com/questions/245453/how-to-not-show-path-after-cd-command-with-cdpath-set )

**Comment by jhfrontz** [comment]
From the `cd` section of `bash` man page:

>>>
If a  non-empty directory  name  from CDPATH is used, or if - is the first argument, and the
directory change is successful, the absolute pathname of the new working directory is written
to the standard  output. 
>>>

**Comment by jhfrontz** [comment]
This allowed me to build/install:

```
$ diff --git a/scripts/pod b/scripts/pod
index 6a35105..61004af 100755
--- a/scripts/pod
+++ b/scripts/pod
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-readonly current_dir="$(cd -P "$(dirname "${BASH_SOURCE[0]}")"; pwd)"
+readonly current_dir="$(cd -P "$(dirname "${BASH_SOURCE[0]}")">/dev/null; pwd)"
 
 exec "$current_dir/exec_with_gems" pod "$@"
```

**Comment by niw** [comment]
Ah, I see. That's unexpected. Let me fix the script, thanks!

**Comment by niw** [comment]
Fixed by `8fafbc6`. Thanks!
